docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3"
+services:
+  mqtt2prometheus:
+    build: .
+    image: stephens/mqtt2prometheus:latest
```

Comment:
Could you please switch here also the image `ghcr.io/hikhvar/mqtt2prometheus:latest` ?